.github/workflows/lint.yml

```diff
@@ -0,0 +1,30 @@
+name: Lints
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    # Setup nodejs and build the webui so that we can 
+    # compile the go code that embeds the webui.
+    - name: Setup node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 21
+    - run: cd webui && npm install
+    - run: cd webui && npm run build
+
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '1.20.6'
```

Comment:
As above.

Reply:
When I didn't specify this I got 1.21.x and a complaint asking me to use `go work use`